sdk/repo/verify.go

```diff
@@ -162,6 +162,38 @@ func (r *repo) projectBranches(p Project) ([]string, error) {
 	return branches, nil
 }
 
+func (r *repo) projectTags(p Project) ([]string, error) {
+	git := exec.Command("git", "show", "-s", "--pretty=%D", "HEAD")
+	git.Dir = filepath.Join(r.root, p.Path)
+	git.Stderr = os.Stderr
+	out, err := git.Output()
+	if err != nil {
+		return nil, err
+	}
+
+	var tags []string
+
+	for _, val := range strings.Split(string(out), ",") {
+		ref := strings.TrimSpace(val)
+
+		if strings.HasPrefix(ref, "tag:") {
+			tagName := strings.Split(ref, " ")[1]
+			tags = append(tags, tagName)
+			continue
+		}
+
+		// In case "m/refs/tags/tagname" exists besides "refs/tags/tagname", handle any prefix
+		if strings.Contains(ref, "refs/tags/") {
+			tagName := strings.Split(ref, "refs/tags/")[1]
+			// The above breaks tags that itself include refs/tags/ but such tags are not expected
+			tags = append(tags, tagName)
+			continue
```

Comment:
This continue is weird... Are you expecting to add more conditional clauses to the loop in the future?